.github/workflows/test.yml

```
@@ -221,18 +239,21 @@ jobs:
            COVERALLS=1 CONTAINER_RUNNABLE=1 make check
         popd
 
-    - name: Build docker image - from current pr branch
+    - name: Build multi-arch docker image - from current pr branch
       if: steps.is_pr_image_build_needed.outputs.PR_IMAGE_RESTORED != 'true' && success()
       run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:12:32: Double quote to prevent globbing and word splitting

(shellcheck)